lib/utils.pm

```diff
@@ -720,7 +720,7 @@ that the boot partition is encrypted.
 
 =cut
 sub is_boot_encrypted {
-    return 0 if get_var('UNENCRYPTED_BOOT');
+    return 0 if get_var('UNENCRYPTED_BOOT') || get_var('HOME_ENCRYPTED');
```

Comment:
why we need this change? I'm not sure ...because we are fixing the modules schedules, also is not documented in variables.md either added to schedule in `vars:` section.

Reply:
FYI i have remove it and use the `UNENCRYPTED_BOOT`